x-pack/plugin/eql/src/test/resources/valid-queries.eql

```diff
@@ -0,0 +1,332 @@
+process where process_name == "svchost.exe" and command_line != "* -k *";
+process where process_name in ('ipconfig.exe', 'netstat.exe', 'systeminfo.exe', 'route.exe');
+process where subtype.create and wildcard(command_line, "*.ost *", "*.pst *")
+;
```

Comment:
Is this file auto-generated?
I ask because it's fairly inconsistent.
👍 on comments.
If there are different types of queries, consider breaking it down.

Reply:
yeah, it's at least partially autogenerated. most of it is copied from the unit tests for the python library. I can definitely clean it up.

Reply:
Only if you think it's a good final state.
Btw, something that we do for docs is to include actual code snippets that are tested (for example: https://github.com/elastic/elasticsearch/blob/4a8e5ada231f879ce0b55276694ee9dc0abf3e76/x-pack/plugin/sql/qa/src/main/resources/docs/docs.csv-spec#L14) - this has the nice side-effect of being always consistent and correct.

Just something to keep in mind (if you have queries that are used in the docs, you can put them aside for the future).